apps/prairielearn/src/pages/instructorQuestionStatistics/instructorQuestionStatistics.js

```diff
@@ -17,7 +17,10 @@ const setFilenames = function (locals) {
 
 router.get(
   '/',
-  asyncHandler(async (req, res) => {
+  asyncHandler(async (req, res, next) => {
+    if (res.locals.question.course_id !== res.locals.course.id) {
```

Comment:
Comment indicating why we have this check?

```suggestion
    // Forbid access to question statistics if question is shared from
    // another course.
    if (res.locals.question.course_id !== res.locals.course.id) {
```

Ditto later in this file.

Reply:
Sure. I think I'll have them be TODO comments because this is not meant to be permanent, we should be supporting looking at the stats for a questoin's use in the course that is consuming it.